newrelic-agent/src/main/java/com/newrelic/api/agent/NewRelicApiImplementation.java

```diff
@@ -147,6 +163,21 @@ public void noticeError(String message, boolean expected) {
         noticeError(message, params, expected);
     }
 
+    private boolean isExpectedErrorConfigured(String expectedConfigName, String message) {
+        boolean expected = true;
```

Comment:
There's really no reason for this `expected` variable to exist as the value is never modified. You could just directly return `true` or `false` in place of it.